maintainers/maintainer-list.nix

```diff
@@ -15755,6 +15755,15 @@
       fingerprint = "7573 56D7 79BB B888 773E  415E 736C CDF9 EF51 BD97";
     }];
   };
+  r17x = {
+    email = "hi@rin.rocks";
+    github = "r17x";
+    githubId = 16365952;
+    name = "Rin";
+    keys = [{
+      fingerprint = "476A F55D 6378 F878 0709  848A 18F9 F516 1CC0 576C";
+    }];
+  };
```

Comment:
I think this is supposed to go in a separate PR? Or is that old information I have?

Reply:
Same PR is fine, but it should be a separate commit called "maintainers: add r17x` since we try to keep a atomic commit history in nixpkgs

Reply:
Done @Janik-Haag

Reply:
I quit nixpkgs and won't be reviewing this any further #323901

Reply:
Thank for your contributions  @Janik-Haag